requirements-doc.txt

```diff
@@ -4,3 +4,4 @@ sphinx>=4  # improved support for docutils>=0.17
 sphinx_rtd_theme>=1  # <1 does not work with docutils>=0.17
 sphinx-gallery
 sphinx-copybutton
+linkml-runtime==1.5.0
```

Comment:
```suggestion
```
Similarly to the above, this is just for any libraries that need to be installed for docs to run that are not installed by `requirements.txt` or `pyproject.toml`

Reply:
Actually, never mind. I think it should not be in `requirements.txt`. Therefore, it should be here.